Migrated to latest pluto release

jeluard opened the pull request:
fixes #7363 

### Summary:

Introduces some new extensions syntax from latest pluto.

_NOTE to testers_ It would make sense to validate with all existing extensions listed [here](https://status.im/extensions/examples.html)

status: ready

jeluard committed:
Migrated to latest pluto release

Signed-off-by: Andrey Shovkoplyas <motor4ik@gmail.com>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | c774d3bf | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7740/1/) | 2019-03-14 16:06:10 | ~3 min | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7740/1/consoleText) |

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | c774d3bf | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7740/1/) | 2019-03-14 16:06:10 | ~3 min | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7740/1/consoleText) |
| :x: | c774d3bf | [#1](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7740/1/) | 2019-03-14 16:06:13 | ~3 min | `macos` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7740/1/consoleText) |

flexsurfer commented on deps.edn:
```diff
@@ -1,6 +1,6 @@
 {:paths ["components/src" "src" "react-native/src/cljsjs" "resources"]
  :deps  {org.clojure/clojure         {:mvn/version "1.9.0"} ;; Keep in sync with .TOOLVERSIONS
-         org.clojure/clojurescript   {:mvn/version "1.10.439"}
+         org.clojure/clojurescript   {:mvn/version "1.10.238"}
```
why downgrade?

jeluard replied:
Somehow we used 2 different versions and there are some issues with using the more recent version.

flexsurfer commented on src/status_im/ui/screens/extensions/add/views.cljs:
```diff
@@ -45,25 +39,25 @@
             [cartouche {:header (i18n/label :t/identifier)}
              [react/text {:style styles/text}
-              (str (get-in data ['meta :name]))]]
+              (str (get-in data [:meta :name]))]]
```
is it still `meta` symbol in extension? or you decided to change symbols on keywords in edn?

jeluard replied:
In the parsed extension it is keywords. The extension syntax didn't change.

Serhy commented:
@jeluard 
1) Wallet related extensions could be installed but they are not shown in Wallet properties after that. 
**To reproduce:**
* Open Status and create new account
* Open any of the https://status.im/extensions/play.html?hash=QmYnUj7v3UiP6X1YfRuhea5mXpjTGG1KKnCwFS4TKzKTpD or https://status.im/extensions/play.html?hash=QmXEmvhNSvAabQ8t7WX4Fz8SJ3KNh8mtLtK3985sYccjmA pages in external browser
* Add extention via QR code scan
* Once success -> navigate to `Wallet` -> top-right menu button to check whether installed extention is present 

It's all fine with nightly develop build

Seems that's only issue with this PR. 
I understand as it mentioned on https://status.im/extensions/examples.html that 'Most of those extensions are not working', but just in case, one issue with "Aragon DAO Voting Plugin" PR (I can't confirm the same extension on develop build because it ends up with `missing-property :on-success` upon install). 
**Steps to reproduce:**
* Open Status and create new account
* Install "Aragon DAO voting plugin" ( https://status.im/extensions/play.html?hash=QmXzaJygxtbYu4ttJ2WpyczuKUrYahiuNT6VV79Dswhi89 )
* Navigate to 1-1 chat and send `/get-events` / `get-events 1` message
**Actual result:**  `null is not an object... ` error appears.
<img width="266" alt="Screenshot 2019-03-18 at 17 25 27" src="https://user-images.githubusercontent.com/8749671/54541616-ed85a880-49a2-11e9-8c98-08be21476483.png">

jeluard commented:
@Serhy Thanks! I fixed the wallet issue.
I also fixed the Aragon extension (updated in the examples page).

There is still some issue to be fixed but I will address it in a separate PR.

Serhy commented:
Looks good to me now. 
Thanks for confirming second issue above (https://github.com/status-im/status-react/pull/7740#issuecomment-473960143) is Dapp specific.
Failed autotest should be fixed in the scope of https://github.com/status-im/status-react/issues/7750 (manually all good, - I used ETHro from another account to verify recovery phrase on Wallet screen)
